### PR TITLE
fixed syntax errors in intro tutorial js

### PR DESCRIPTION
**Pull request by Benjamin28**
**Description:**

**Changes proposed:**
-change errant ; to ,
-add ; at end of line

### CODE REVIEW

File: docs/guides/index.md [hunk 1]
```diff
@@ -527,15 +527,16 @@ Events][animatingonevents] below.
 
 In JavaScript, we grab the box with [`querySelector`][queryselector], use
 [`addEventListener`][addeventlistener], and then [`setAttribute`][setattribute]
-to make the box grow its scale when its hovered over. Note that A-Frame adds
+to make the box grow its scale when its hovered over.The code must appear after
+the scene so that the box has loaded when the script runs. Note that A-Frame adds
```
Comment (dmarcos) [inline docs/guides/index.md]: You left the docs changes.

Comment (Benjamin28) [inline docs/guides/index.md]: There are 2 separate commits right?

Comment (Benjamin28) [inline docs/guides/index.md]: I'll undo the second commit.

Comment (dmarcos) [inline docs/guides/index.md]: I meant. Two PRs: one with only the typo fix and the other with the doc clarification

**Comment by Benjamin28** [comment]
Thanks for your patience. This is my first commit to a big open source project.

**Comment by dmarcos** [comment]
Nice! Thanks!

**Pull request merged**